docs/config/README.md

```diff
@@ -50,6 +50,13 @@ minio server ~/Photos
 |:---|:---|:---|
 |``region``| _string_ | `region` describes the physical location of the server. By default it is set to `us-east-1`, which is same as AWS3's default region. If you are unsure leave it to default.|
 
```

Comment:
The environment variable also needs to be mentioned here (in the table, like for MINIO_ACCESS_KEY)